packages/core/src/components/header/test/default/header.e2e.ts

```diff
@@ -0,0 +1,35 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/header/test/default/index.html';
+
+test.describe('tds-header-default', () => {
+  test('renders default header correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponent = page.getByRole('navigation');
+    await expect(headerComponent).toHaveCount(1);
+
+    /** Check screenshot diff */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('Header title exists and is "Example: default"', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponentHeaderText = page.getByText('Example: default');
+    await expect(headerComponentHeaderText).toBeVisible();
+  });
+
+  test('Header luncher icon exists', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponentLuncherIcon = page.getByRole('img');
```

Comment:
Could this also select the brand icon?
Perhaps it would be good to combine this locator with another, like the button label or a test-id

Reply:
It can not since brand icon is an css background-image

Reply:
updated: [2d0bc34](https://github.com/scania-digital-design-system/tegel/pull/461/commits/2d0bc348527c6ace24db0439ac34cbdf3c0d206b)